### PR TITLE
added better seo

### PR DESCRIPTION
**Pull request by rishabhsharma1997**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -4,15 +4,16 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-
+    <meta name="test" content="testing" />
```
Comment (leecalcote) [inline _includes/header.html:7]: Is this a work in-progress?

**Comment by Ashparshp** [comment]
@rishabhsharma1997 Any luck?

**Comment by sudhanshutech** [comment]
@rishabhsharma1997 is this completed?

**Comment by shubhusion** [comment]
this PR is related to which issue ?

File: _includes/header.html (hunk 1 above)
Comment (leecalcote) [reply _includes/header.html:7]: @amitamrutiya, do you think that you can assist @rishabhsharma1997 in finishing off this work?

Comment (amitamrutiya) [reply _includes/header.html:7]: Sure 💯

**Comment by amitamrutiya** [comment]
Hi @rishabhsharma1997, just checking in—has this been completed?

**Comment by sudhanshutech** [comment]
@rishabhsharma1997 checking again

**Review by captain-Akshay — APPROVED** [review]

**Pull request merged**